examples/2d/bar_waves/step01_pwave_reflected.cfg

```
@@ -56,7 +56,7 @@ bc.bc_domain = pylith.bc.DirichletTimeDependent
 
 [pylithapp.problem.bc.bc_xneg]
 constrained_dof = [0]
-label = boundary_xneg
+label = boundary_xneg 
```

```suggestion
label = boundary_xneg
```
Drop trailing whitespace.